example/index.html

```diff
@@ -49,56 +54,71 @@
           debugLoggingEnabled: true,
           gatewayUrl: 'https://youraccount-test.testonly.conviva.com', // TOUCHSTONE_SERVICE_URL for testing
-          custom: {
-            directors: 'Test director 1, Test director 2',
-            year: '2010',
-          },
-          additionalStandardTags: {
-            'c3.cm.brand': 'Test Brand',
-          }
+          // This is inferred automatically, but can be overridden.
+          // deviceMetadata: { ... }
         });
       }
 
       function loadPlayer() {
-        console.log('Updating Conviva content metadata');
-        conviva.updateContentMetadata({
-          applicationName: 'Bitmovin Player Conviva Analytics Integration Test Page',
-          viewerId: 'uniqueViewerId',
+        console.log('Updating Conviva Analytics content metadata');
+        // Initialize metadata. It also can be used to update some data during playback.
+        convivaAnalytics.updateContentMetadata({
+          viewerId: 'bitmovin-player-web-conviva-integration-viewer',
+          // It's also referred as "Player name" on Conviva in some places.
+          applicationName: 'Bitmovin Web Player',
+          assetName: 'Art of Motion',
+          defaultResource: 'TEST-DEFAULT-RESOURCE',
           custom: {
-            appVersion: '1.0',
-            contentId: 'someContentId',
+            directors: 'Director 1, Director 2',
+          },
+          // Reserved Conviva tags can be placed here.
+          // Find the list:
+          // - Here https://pulse.conviva.com/app/appmanager/meta-data (Required Metadata tab)
+          // - Or here https://pulse.conviva.com/learning-center/content/sensor_developer_center/sensor_integration/javascript/js_quick_integration.htm
+          //   under "Constants for Pre-defined Video and Content Metadata"
+          additionalStandardTags: {
+            'c3.app.version': '0.0.1',
+            'c3.cm.contentType': 'VOD',
+            'c3.cm.channel': 'Test Channel',
+            'c3.cm.brand': 'Test Brand',
+            'c3.cm.affiliate': 'Test Affiliate',
+            'c3.cm.categoryType': 'Test Category',
+            'c3.cm.name': 'Test CM Name',
+            'c3.cm.id': 'test-cm-id',
+            'c3.cm.seriesName': 'Test Series Name',
+            'c3.cm.seasonNumber': 'Test Season Number',
+            'c3.cm.showTitle': 'Test Show Title',
+            'c3.cm.episodeNumber': 'Test Episode Number',
+            'c3.cm.genre': 'Test Primary Genre',
+            'c3.cm.genreList': 'Test Genre List Item 1, Test Genre List Item 2',
+            'c3.cm.utmTrackingUrl': 'https://test-utm-tracking-url',
+            // Cannot be determined reliably, but if there is a need this approximation can be used.
+            // dcType: (navigator as Navigator & { connection?: { type: string } }).connection?.type
+            //   || (navigator as Navigator & { connection?: { effectiveType: string } }).connection?.effectiveType
```

Comment:
We should make sure this is valid Javascript (not Typescript) in here

Reply:
Right https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/109/commits/4a138baac41b8ddcbf4c894732f434a4c945652d. 🙂